core/flyout_nav.js

```diff
@@ -0,0 +1,543 @@
+'use strict';
+
+goog.provide('Blockly.FlyoutNav');
+
+
+
+/*
+*Licensed under the Apache License, Version 2.0 (the "License");
+*you may not use this file except in compliance with the License.
+*You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing, software
+*distributed under the License is distributed on an "AS IS" BASIS,
+*WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*See the License for the specific language governing permissions and
+*limitations under the License.
+*/
+
+/*
+* File overview: this file handles using the keyboard to navigate the flyout
+*/
```

Comment:
All the other js files in blockly generally include the full header with the license as the first thing at the top of the file, might be a good idea to follow that pattern.